### DISCUSSION TITLE
Use GitHub page domain for vercel apps

### DISCUSSION THREAD

**Discussion by mohammadius** [vercel/community#834] [Help] [answered]
I'm creating a personal website with next.js and planning to host it on vercel. I'm wondering if it's possible to use my `username.github.io` domain as a custom domain for the vercel app, without redirecting.
I know it's possible with redirecting to the `.vercel.app` domain but I want it to be like a "real" custom domain.
`username.vercel.app` is ok and I still want that to be accessible but I want to also use the website for my GitHub projects pages. so having both domain would be nice.

**Comment by swarnava** [comment, marked as answer]
Hi,

If you host a Project on Vercel (without GitHub pages), you won't be able to setup .github.io domain. GitHub doesn't allow to use their domain to serve as custom domain.
